test/appsignal/span_test.exs

```diff
@@ -270,8 +270,7 @@ defmodule AppsignalSpanTest do
     end
 
     test "sets the error through the Nif", %{span: %Span{reference: reference}} do
-      assert [{^reference, "ArgumentError", "** (ArgumentError) argument error", _}] =
-               Test.Nif.get!(:add_span_error)
+      assert [{^reference, "ArgumentError", _message, _}] = Test.Nif.get!(:add_span_error)
```

Comment:
I think this is fine. The "ArgumentError" is enough for what we want to test. Maybe we only want to know _if there is a message at all_. Does this check if `_message` has any value?

Reply:
Yes, but it ignores the value check

Reply:
It doesn't look like there any special behavior for `ArgumentError`, I think we can omit it from this test. If a message is set for the error is also asserted in other tests.